awwvision/src/main/java/com/google/cloud/servicebroker/samples/awwvision/service/ImageLabelingService.java

```diff
@@ -29,10 +29,10 @@
 import java.io.IOException;
 
 /**
- * Helper methods for interacting with the Cloud Vision API.
+ * Component which labels images using Google Vision.
```

Comment:
I think our branding guidelines say the product name needs to be "Cloud Vision API"